src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -436,7 +436,8 @@ class GuiPluginList extends React.PureComponent<Props, State> {
             ListHeaderComponent={this.renderTop}
             renderItem={this.renderPlugin}
             keyExtractor={(item: GuiPluginRow) => item.pluginId + item.title}
-            contentContainerStyle={insetStyle}
+            // XXX: Hack. paddingBottom from insetStyle is not sufficient.
+            contentContainerStyle={{ ...insetStyle, paddingBottom: theme.rem(6) }}
```

Comment:
Why is it not sufficient? What's causing this?

Reply:
I don't know but I'm NOT touching common code in the SceneWrapper this late in the game.